addons/account/models/res_partner_bank.py

```diff
@@ -55,7 +55,7 @@ def _build_qr_code_vals(self, amount, free_communication, structured_communicati
         available_qr_methods = self.get_available_qr_methods_in_sequence()
         candidate_methods = qr_method and [(qr_method, dict(available_qr_methods)[qr_method])] or available_qr_methods
         for candidate_method, candidate_name in candidate_methods:
-            if self._eligible_for_qr_code(candidate_method, debtor_partner, currency):
+            if self._eligible_for_qr_code(candidate_method, debtor_partner, currency, not silent_errors):
```

Comment:
I think it could be better to adjust checks done in `_eligible_for_qr_code` instead of suppressing errors maybe?

Reply:
I had addressed and discussed other alternative solutions, including adjusting checks done in `_eligible_for_qr_code`, with Camille(casp). She suggested that it was better to suppress the error when the qr code is being generated for only for payments aka when it's called by _`compute_qr_code`. 

Based on the fact that the compute method did not explicitly pass in the silent_errors arg into `build_qr_code_base64`, I assumed that the errors should be suppressed by default since the definition of both `build_qr_code_base64` and` _build_qr_code_vals` has `silent_errors=True`. 

Thus, I thought it was just appropriate for the raises_error param of `_eligible_for_qr_code` to be consistent with the `silent_errors` param.

Reply:
Also, I was unable to reproduce the issue on Runbot 14 and on Runbot 15, unless you were able to do so?

Reply:
Sorry, the problem indeed exists from 16.0, I misunderstood something :slightly_smiling_face: